DEVELOP.rst

```diff
@@ -174,9 +174,84 @@ Before creating a new distribution, a new version and tag should be created:
 
 - Update ``CURRENT`` in ``io.crate.Version``
 
-- Add a section for the new version in the ``CHANGES.txt`` file
+- Prepare the release notes
 
-- Commit your changes with a message like "prepare release x.y.z"
+  If a technical writer is available, ask them to complete these steps.
```

Comment:
i think it is best to not define any responsibilities here, just tell what needs to be done